discussions/tasks.py

```diff
@@ -129,6 +131,32 @@ def add_user_as_moderator_to_channel(user_id, program_id):
         )
 
 
+@app.task()
+def remove_moderators_from_channel(channel_name):
+    """
+    Remove moderators of a given channel_name.
+    Args:
+        channel_name(str): channel name
+    """
+
+    mod_ids = Role.objects.filter(
```

Comment:
Can you refactor this and the usage in the  other method to a `get_moderators_for_channel` function?